structset_test.go

```diff
@@ -38,6 +38,27 @@ func TestStructset(t *testing.T) {
 			Name: "Luffy",
 		}
 		doc      = NewDocument(&user)
+		mutation = Mutation{
+			Cascade: true,
+			Mutates: map[string]Mutate{
+				"id":         Set("id", 1),
```

Comment:
Existing tests have `ID` field explicitly set, so after I've implemented my change they are failing. I've added `Set` mutation to list of expected mutations, but I can also remove value for `ID` field of the structure.